core/services/job/orm.go

```diff
@@ -1179,6 +1182,32 @@ INNER JOIN ccip_specs ccip on jobs.ccip_spec_id = ccip.id AND ccip.capability_la
 	return
 }
 
+func (o *orm) FindStandardCapabilityJobID(ctx context.Context, spec StandardCapabilitiesSpec) (jobID int32, err error) {
+	stmt := `
+SELECT jobs.id FROM jobs
+INNER JOIN standardcapabilities_specs sc on jobs.standard_capabilities_spec_id = sc.id AND sc.command = $1
+`
+	err = o.ds.GetContext(ctx, &jobID, stmt, spec.Command)
+	if err != nil && !errors.Is(err, sql.ErrNoRows) {
+		err = fmt.Errorf("error searching for job for standardcapabilities (command) ('%s'): %w", spec.Command, err)
+	}
+	return
+}
+
+func (o *orm) FindGatewayJobID(ctx context.Context, spec GatewaySpec) (jobID int32, err error) {
```

Comment:
@Tofel Hmmmm... I think you're overthinking this; we can probably just use the name of the gateway here (that also has the benefit of removing a bunch of the extra methods you added above)